draft-ietf-quic-tls.md

```diff
@@ -113,17 +113,19 @@ connections between the same client and server, the client can often
 send application data immediately, that is, using a zero round trip
 setup.
 
-This document describes how the standardized TLS 1.3 can act a security
+This document describes how the standardized TLS 1.3 acts a security
 component of QUIC.  The same design could work for TLS 1.2, though few of the
 benefits QUIC provides would be realized due to the handshake latency in
 versions of TLS prior to 1.3.
 
 
 # Notational Conventions
 
-The words "MUST", "MUST NOT", "SHOULD", and "MAY" are used in this document.
-It's not shouting; when they are capitalized, they have the special meaning
-defined in {{!RFC2119}}.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED",
+"MAY", and "OPTIONAL" in this document are to be interpreted as
+described in BCP 14 {{!RFC2119}}, {{!I-D.leiba-rfc2119-update}} when, and only when, they
+appear in all capitals, as shown here.
```

Comment:
Does this mean you don't want this change, or you are just annoyed that there is a draft on this?